client/src/components/styles/ContainerReviews.styles.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
```

Comment:
You can use props to re-use this component with container average stars. EG:
<Wrapper review>
Wrapper = styled.div`
align-items: ${props => props.reviews ? 'center' : ''}
`

Reply:
If I could paste a fire emoji here, I would.  Thank you!  This is awesome feedback!